packages/animation-curve-block/src/components/Card.tsx

```diff
@@ -87,7 +88,11 @@ export const Card = forwardRef<HTMLDivElement, CardProps>(
                     shouldHideWrapper={replaceWithPlaceholder || !isEditing}
                     shouldHideComponent={replaceWithPlaceholder}
                     toolbarItems={[
-                        { icon: <IconArrowMove16 />, tooltip: 'Drag to move', draggableProps },
+                        {
+                            icon: <IconArrowMove16 />,
```

Comment:
you could even move this one to shared as it is everywhere the same Icon

Reply:
won't do that as the icon and tooltip then could be optionally but it isn't in the other case